### ISSUE TITLE
Namespace OurUtils doesn't exist in the namespace GooglePlayGames

### ISSUE THREAD

**Issue by satanas** [playgameservices/play-games-plugin-for-unity#1014]
Hello guys,

I'm getting this error when switching platform from Android  to iOS with the `NO_GPGS` symbol defined:
```
Assets/GooglePlayGames/Platforms/Native/PInvoke/BaseReferenceHolder.cs(24,27): error CS0234: The type or namespace name `OurUtils' does not exist in the namespace `GooglePlayGames'. Are you missing an assembly reference?
```

In Android the plugin works like a charm but once I switch platform to use GameCenter (through the Unity Social API) I get the error mentioned above.

I just deleted manually Assets/Plugins/iOS/GPGSAppController.h and .mm but still doesn't work. I'm using v0.9.31.

Any clue?

Thanks

**Comment by claywilkinson** [comment]
This was fixed in 0.9.31a.  You can also refer to https://github.com/playgameservices/play-games-plugin-for-unity/blob/master/source/PluginDev/Assets/GooglePlayGames/Platforms/Native/PInvoke/BaseReferenceHolder.cs to get the change to the #ifdef line that excludes this file when NO_GPGS is defined.

**Comment by satanas** [comment]
Hell yeah! Now it's working (using 0.9.31a).

Thanks for your support :+1:
